Open_Source_Events/NWOC/README.md

```diff
@@ -0,0 +1,13 @@
+<h1 align="center"> NJACK Winter of Code 2020</h1>
+
+<h2> About NWOC'20 </h2>
+
+NJACK Winter of Code is an initiative by NJACK, IIT Patna, targeted at students who have never participated in Free or Open Source Software (FOSS) development before,
+as well as at those who have already become an expert in Open Source Development and are currently contributing to projects of any domain, say Web, Mobile, Machine Learning,
+Blockchain, IoT etc. It also aims to prepare them for bigger programs and opportunities like Google Summer of Code, Outreachy etc.
```

Comment:
Don't use words like `etc`. You can use alternatives.

Reply:
Sure !!